README.md

```diff
@@ -5,8 +5,8 @@
 ![screenshot](./src/appScreenshot.png)
 
 
-- [Live Demo](https://stock-metrics-clint.netlify.app/)
-- [Presentation Video](https://www.loom.com/share/eba6bdff52dc4a3a805a3a98a7a7064c)
+- [Live Demo](https://themetricshub.netlify.app/)
+- [Presentation Video](https://drive.google.com/file/d/19mlq_eMxa6t21e6JeOM0OYQiVmqzujHV/view?usp=sharing)
```

Comment:
### GOOD JOB 👏🎉😃

Everything is working as expected 📱. Thank you also for the video presentation 🎥. I have never seen any student added real company logo to the details in this project. Keep on creating such an awesome apps 👍